requirements-dev.txt

```diff
@@ -8,3 +8,5 @@ click>=7.0
 ruamel.yaml>=0.15.89
 pytest==5.3.4
 pytest-subtests==0.3.0
+pytest-cov==2.8.1
+pymongo==3.10.1
```

Comment:
We are actually trying to move away from the optional pymongo integration, but I guess we can always drop that at some later point.

Reply:
`pymongo` is required for the `config host` command. This can be removed after the its deprecation cycle.